application/src/main/java/org/opentripplanner/service/osminfo/internal/DefaultOsmInfoGraphBuildRepository.java

```diff
@@ -8,12 +8,15 @@
 import java.util.Optional;
 import org.opentripplanner.service.osminfo.OsmInfoGraphBuildRepository;
 import org.opentripplanner.service.osminfo.model.Platform;
+import org.opentripplanner.street.model.edge.Area;
 import org.opentripplanner.street.model.edge.Edge;
+import org.opentripplanner.utils.tostring.ToStringBuilder;
 
 public class DefaultOsmInfoGraphBuildRepository
   implements OsmInfoGraphBuildRepository, Serializable {
 
   private final Map<Edge, Platform> platforms = new HashMap<>();
+  private final Map<Area, Platform> areaPlatforms = new HashMap<>();
```

Comment:
Area doesn't seem to have equals or hashcode. Are the maps using idenity in that case?

Reply:
I did not even realize this issue, just trusted Java magic. I have to check this.

Reply:
I guess java uses object address as hash or something like that. I do nmot see any problem in it.